python/cugraph-pyg/cugraph_pyg/sampler/cugraph_sampler.py

```diff
@@ -36,6 +36,48 @@
 )
 
 
+def _sampler_output_from_sampling_results(
+    sampling_results: cudf.DataFrame,
+    graph_store: CuGraphStore,
+    metadata: Sequence = None,
+):
```

Comment:
Can you add the return type annotation too?

Reply:
fixed